src/browsing.md

```diff
@@ -220,9 +223,9 @@ they can be found in a submenu of the context menu.
 | Name | Action |
 |-|-|
 | Change Deck | Move currently selected cards to a different deck. |
-| Set Due Date | Turn cards into review cards, and make them due on a certain date. This can be useful for moving cards forward or back a few days when your study schedule is interrupted. Entering a range like `60-90` will give cards an interval between 60 and 90 days, which can be useful when you've imported a bunch of material you already know.
+| Set Due Date | Turn cards into review cards, and make them due on a certain date. This can be useful for moving cards forward or back a few days when your study schedule is interrupted. Entering a range like `60-90` will give new cards an interval between 60 and 90 days, which can be useful when you've imported a bunch of material you already know.
```

Comment:
Not sure if I got this right, but I think only new cards receive the interval from Set Due Date, whereas an existing interval won't be modified. Is that correct?

Reply:
```suggestion
| Set Due Date | Turn cards into review cards, and make them due on a certain date. This can be useful for moving cards forward or back a few days when your study schedule is interrupted. Entering a range like `60-90` will make the selected cards due between 60 and 90 days from now. New cards will have their interval set to the same delay, but reviews will be rescheduled without changing their current interval, unless '!' is included at the end of the range.
```

Reply:
Yep, but it might be worth mentioning that reviews are still rescheduled